wordpress-mobile/WordPress-iOS
# Change ReaderPost `card` relationship from "One to many" to "To one"

This PR changes the `ReaderPost` `card` relationship from "One to many" to "To one".

### To test

First, install an earlier version of the app (not this branch).

1. Run the older version
2. Go to Discover
3. Save a post

Switch to this branch:

1. Run the app
2. Go to the "Saved" tab
3. Make sure the post is there
4. Interact with the post
5. Unsave it
6. Go to the Discover tab

## Regression Notes
1. Potential unintended areas of impact
The whole app, it's a Core Data migration. :)

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Relied on unit tests migration and manually testing it.

3. What automated tests I added (or what prevented me from doing so)
-

PR submission checklist:
